### PR TITLE
Revert "fix: quote all tables/columns/indexes in postgresql schema script (#5)

### PR DESCRIPTION
**Pull request by rpignolet**
This reverts commit afd18ae17cd7593973dedfa6fe89cf7056d60f48.

Revert the PR https://github.com/TOSIT-IO/hive-old/pull/5

### CODE REVIEW

File: standalone-metastore/src/main/sql/postgres/hive-schema-3.1.0.postgres.sql [hunk 1]
```diff
@@ -1820,32 +1820,32 @@ CREATE TABLE "SCHEMA_VERSION" (
   "SCHEMA_ID" bigint references "I_SCHEMA" ("SCHEMA_ID"),
   "VERSION" integer not null,
   "CREATED_AT" bigint not null,
-  "CD_ID" bigint references "CDS" ("CD_ID"),
+  "CD_ID" bigint references "CDS" ("CD_ID"),
```
Comment (Pierrotws) [inline standalone-metastore/src/main/sql/postgres/hive-schema-3.1.0.postgres.sql]: typo, should not be commited

File: standalone-metastore/src/main/sql/postgres/hive-schema-3.1.0.postgres.sql [hunk 2]
```diff
@@ -1820,32 +1820,32 @@ CREATE TABLE "SCHEMA_VERSION" (
...
   "CREATED_AT" bigint not null,
-  "CD_ID" bigint references "CDS" ("CD_ID"),
+  "CD_ID" bigint references "CDS" ("CD_ID"), 
   "STATE" integer not null,
   "DESCRIPTION" varchar(4000),
   "SCHEMA_TEXT" text,
   "FINGERPRINT" varchar(256),
   "SCHEMA_VERSION_NAME" varchar(256),
-  "SERDE_ID" bigint references "SERDES" ("SERDE_ID"),
+  "SERDE_ID" bigint references "SERDES" ("SERDE_ID"),
```
Comment (Pierrotws) [inline standalone-metastore/src/main/sql/postgres/hive-schema-3.1.0.postgres.sql]: typo, should not be commited

**Review by Pierrotws — COMMENTED** [review]
OK, but we might consider removing initial commit from branch 3.1.3

**Comment by Pierrotws** [comment]
Considering that this branch is already not the one that has built initial tosit 3.1.3-1.0, shouldn't we force remove initial commit from branch, instead of committing a revert ?

File: standalone-metastore/src/main/sql/postgres/hive-schema-3.1.0.postgres.sql (hunk 1 above)
Comment (rpignolet) [reply standalone-metastore/src/main/sql/postgres/hive-schema-3.1.0.postgres.sql]: This is a revert commit.

File: standalone-metastore/src/main/sql/postgres/hive-schema-3.1.0.postgres.sql (hunk 2 above)
Comment (rpignolet) [reply standalone-metastore/src/main/sql/postgres/hive-schema-3.1.0.postgres.sql]: This is a revert commit.

**Comment by rpignolet** [comment]
> Considering that this branch is already not the one that has built initial tosit 3.1.3-1.0, shouldn't we force remove initial commit from branch, instead of committing a revert ?

Yes we can, for now this is for testing purpose.

**Review by mehdibn — APPROVED** [review]

**Comment by rpignolet** [comment]
Commit afd18ae17cd7593973dedfa6fe89cf7056d60f48 deleted. Branch 3.1.3-TDP force pushed.

**Pull request closed without merging**